src/libs/ajax/DataRepo.ts

```diff
@@ -63,16 +64,22 @@ export type SnapshotBuilderFeatureValueGroup = {
   values: string[];
 };
 
-export type SnapshotBuilderDatasetConceptSets = {
-  name: string;
+export interface DomainConceptSet extends DatasetBuilderType {
+  concept: SnapshotBuilderConcept;
   featureValueGroupName: string;
-};
+}
+
+export interface PrepackagedConceptSet extends DatasetBuilderType {
+  featureValueGroupName: string;
+}
+
+export type ConceptSet = DomainConceptSet | PrepackagedConceptSet;
```

Comment:
Might I suggest making `ConceptSet` a supertype of `DomainConceptSet` and `PrepackagedConceptSet`, rather than a union? Then you could do:

```
export interface ConceptSet extends DatasetBuilderType {
  featureValueGroupName: string;
}
```
and
```
export interface DomainConceptSet extends ConceptSet {
  concept: SnapshotBuilderConcept;
}
```
The advantage there is you avoid duplication of `featureValueGroupName`

Reply:
I like this idea but then I would lose the individual PrepackagedConceptSet type, right? I kind of liked the clarity that that brought. What are your thoughts on this?

Reply:
You could just add the type as:
`export interface PrepackagedConceptSet extends ConceptSet {}` (I think)

Reply:
Okay awesome thank you!